docs/fundamentals/dashboard/configuration.md

```diff
@@ -0,0 +1,114 @@
+---
+title: .NET Aspire dashboard configuration
```

Comment:
This should mention `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL`. I had to go digging into the code to find that env.